private/manageFishData.py

```diff
@@ -429,6 +454,10 @@ def check_data_integrity(args):
     return not has_errors
 
 
+def createSprites():
```

Comment:
I'd rather not include this in the Python script.  Technically speaking, the individual images this script generates are all it needs to know about making.  Uhh, I see it as independent of the actual site's design (aside from it generating the JS file).  I would rather have build steps related to deployment managed by node if possible, so it would call this, and then run the task to generate the sprite map if new images were detected.

Reply:
Ahh ok then we can remove it here (can't really help with node itself)